ui/page/about_page.go

```diff
@@ -66,14 +66,24 @@ func NewAboutPage(l *load.Load) *AboutPage {
 	return pg
 }
 
+// ID is a unique string that identifies the page and may be used
+// to differentiate this page from other pages.
+// Part of the load.Page interface.
 func (pg *AboutPage) ID() string {
 	return AboutPageID
 }
 
-func (pg *AboutPage) OnResume() {
+// WillAppear is called when the page is about to displayed and may
```

Comment:
```suggestion
// WillAppear is called when the page is about to be displayed and may
```
i think this reoccurs everywhere.. so we can just modify it.